src/System.Threading.Tasks.Dataflow/src/Base/DataflowBlockOptions.cs

```diff
@@ -170,6 +177,13 @@ public string NameFormat
                 _nameFormat = value;
             }
         }
+
+        /// <summary>Gets whether ordered processing should be enforced on a block's handling of messages.</summary>
```

Comment:
I wonder if we should "invert" the option, i.e. `AllowUnorderedProcessing` or something like that? I usually prefer configurable options to default to false, but that might just be a personal preference thing.